Source/WebCore/platform/graphics/filters/FEDisplacementMap.h

```diff
@@ -52,6 +52,8 @@ class FEDisplacementMap final : public FilterEffect {
     float scale() const { return m_scale; }
     bool setScale(float);
 
+    static IntOutsets calculateOutsets(const FloatSize& adjustedScale);
```

Comment:
Can we stick with one name for the argument? Maybe `displacementMaxOffset` or something.